docs/pages/careers/design-engineer-x-grid.md

```diff
@@ -0,0 +1,122 @@
+We envision a future where MUI X becomes the default toolkit for web developers to create advanced UIs.
+It's why we started Base UI, Joy UI, and MUI Toolpad. Design is key to achieving this goal.
+
+## The role
+
+### What you'll do on a day-to-day basis
+
+You will extend the [Design Engineer](https://mui-org.notion.site/Design-Engineer-09e4cac2e0bb4df3b173458b4b4b5666) responsibilities. Depending on the day, you'll:
+
+- **Contribute to deep, meaningful refactors and feature releases**.
+  MUI X is a complex codebase. Components we've shipped recently, such as the data grid and date pickers have required months of dedicated, careful work.
+- **Design for engineers and their end users**.
+  As an engineer yourself, you are able to understand problems developers face, and you will help them overcome those problems by delivering good DX, and enabling them to create powerful UX for their end users
+  This might involve careful API design, identifying and fixing top bugs, creating easier-to-understand error messages, and writing documentation and blog posts about features you and your team ship.
+- **Collaborate with the community**.
+  Many small, as well as meaningful fixes and features, have been contributed by the community. Your role is to draw the best out of the community — to inspire those across the world to create and contribute through your reviews of their issues and pull requests.
+- **Experiment and play**. Great, unexpected features and heisenbug fixes have come from a number of sources — relentlessly methodical processes of elimination, free-flowing team collaboration, inspiration by adjacent libraries and projects, and difficult-to-explain individual strokes of brilliance. Whatever your preferred style is for creating new things that others might not have thought of, you'll find a welcome home on the team.
+- **Take ownership of features from idea/mockup to live deployment**.
+  You'll shape and guide the direction of crucial new features, including new components.
+- **Ship. Early and often**. You'll iterate and ship frequently.
+  You'll have a real impact on the end-user experience and you'll love working on a team that builds stunning UIs and prioritizes delivering real user value as often as possible.
+- **You'll be interacting with the users** on a regular basis, handling inbound support and feature requests (every developer helps with developer requests).
+- **Bridge the gap between design and engineering**.
+  You will closely work with designers and engineers, and act as a communication conduit between the two roles to ensure efficient workflows and shared understanding of goals.
+
+### The best parts of this job
```

Comment:
I'm honestly up for removing both the "The best" and "The worst" sections here. The "the best" part doesn't add a lot; most of it has already been said one way or another. And the "the worst" part is probably better shared during the interviews? 

Or at least, not necessarily framed as "the worst" — they are more like any startup challenge (i.e., context switching, speed vs. quality, complexity, novel problem, etc.).

Reply:
I'm up for a leaner description in general. I don't think we need to oversell ourselves. If we were an unknown company or project then yes but in our case I don't think it's needed.

Reply:
But do we need to say something else here? I mean, there are already the sections above (company, product, culture) that cover these topics somehow. 

Also, adding some more thoughts to "the worst part", I appreciate the transparency in saying things like "it's a complex code, be aware!", "you might get overwhelmed by the OSS repo! are you sure?" etc. But I think there might be a better way to frame them not as "the worst" but more like... challenges? Part of the gig, you know?

Reply:
I agree with rephrasing the 'worst' here 🤔 It's great that we are being transparent, but IMO challenges are not necessarily something to run from or paint as entirely negative. We can rename this section to 'challenges you might face', WDYT?